Update Debian control file to add missing dependencies.

jouillet opened the pull request:
Tracked-On:https://github.com/projectacrn/acrn-hypervisor/issues/6688
Some Debian dependencies are currently missing in the `/debian/control` file.  This change adds these missing dependencies:

**Source: acrn-hypervisor**
**Build-Depends:**
- python3-requests
- python3-tqdm
 
**Package: python3-acrn-board-inspector**
**Depends:**
- usbutils


Note: These changes should be part of the ACRN 3.0 release. Should I open a similar PR to the `release_3.0` branch?

wenlingz commented:
> Some Debian dependencies are currently missing in the `/debian/control` file. This change adds these missing dependencies:
> 
> **Source: acrn-hypervisor** **Build-Depends:**
> 
> * python3-requests
> * python3-tqdm
> 
> **Package: python3-acrn-board-inspector** **Depends:**
> 
> * usbutils
> 
> Note: These changes should be part of the ACRN 3.0 release. Should I open a similar PR to the `release_3.0` branch?

Created in release_3.0: https://github.com/projectacrn/acrn-hypervisor/pull/7814

wenlingz commented:
hi @jouillet  please help add "Tracked-On: https://github.com/projectacrn/acrn-hypervisor/issues/6688" in your commit, thanks.

jouillet commented:
> hi @jouillet please help add "Tracked-On: #6688" in your commit, thanks.

@wenlingz I added a signed commit referencing the Tracked-On issue.

dbkinder commented on debian/control:
```diff
@@ -34,7 +34,9 @@ Build-Depends:
    python3-defusedxml,
    python3-xmlschema (>= 1.10.0),
-   python3-elementpath (>= 2.5.0)
+   python3-elementpath (>= 2.5.0),
```
@junjiemao1 @NanlinXie The elementpath version must be = 2.5.0 and not greater, right?

jouillet replied:
If it needs to be exactly a version, the Debian control file syntax is: `python3-elementpath (= 2.5.0)`

dbkinder replied:
OK (the pip install syntax is with ==).  Thanks for clarifying for the control file.  We'll need @junjiemao1 to confirm the exact version (and I'll update the GSG documentation as well).

junjiemao1 replied:
As this PR is against the main branch, preserving the `>= 2.5.0` is also fine to me. I have a patch to resolve the incompatibility issue so that any version since 2.5.0 (including the latest 2.5.3) will work.

szhen11 commented:
> > hi @jouillet please help add "Tracked-On: #6688" in your commit, thanks.
> 
> @wenlingz I added a signed commit referencing the Tracked-On issue.

@jouillet Please follow https://projectacrn.github.io/latest/developer-guides/contribute_guidelines.html#commit-guidelines to add title and body and Tracked-On: in your commit message for all commits.

jouillet commented:
> > > hi @jouillet please help add "Tracked-On: #6688" in your commit, thanks.
> > 
> > 
> > @wenlingz I added a signed commit referencing the Tracked-On issue.
> 
> @jouillet Please follow https://projectacrn.github.io/latest/developer-guides/contribute_guidelines.html#commit-guidelines to add title and body and Tracked-On: in your commit message for all commits, and resolve the "request changes".

I've rebased to latest and modified the commit message accordingly.
